yarn.lock

```diff
@@ -9249,7 +9249,7 @@ jest-worker@^26.2.1, jest-worker@^26.6.2:
     merge-stream "^2.0.0"
     supports-color "^7.0.0"
 
-jest@^26.5.0, jest@^26.6.3:
+jest@^26.5.0:
```

Comment:
Hmm, is this an intended removal of `jest@^26.6.3`?

Reply:
Good spot, not intentional but this change occurs whenever `yarn` is run, and does so on latest for me too. Will remove this change for now 👍